lighthouse-core/audits/byte-efficiency/offscreen-images.js

```diff
@@ -134,8 +134,8 @@ class OffscreenImages extends ByteEfficiencyAudit {
       });
 
       const headings = [
-        {key: 'preview', itemType: 'thumbnail', text: ''},
-        {key: 'url', itemType: 'url', text: 'URL'},
+        {key: 'preview', itemType: 'thumbnail', text: '', styleAs: 'preview'},
+        {key: 'url', itemType: 'url', text: 'URL', styleAs: 'url'},
```

Comment:
changed to `styleClass` per eric's suggestion and difference is structure of dom (which renderer to use) vs. styles to apply (which css to apply). they will typically move together seemed bad to force them to always be the same